### PR TITLE
Add support for parsing quantity datatype

### PR DESCRIPTION
**Pull request by nelson-liu**
This PR introduces changes to support parsing quantities. See https://www.wikidata.org/wiki/Help:Data_type#Quantity for official wikidata docs

I've been using the unitless quantity at https://www.wikidata.org/wiki/Q605704 for testing, as well as the unit-bearing (and uncertain) quantity at https://www.wikidata.org/wiki/Q520.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ To be released.
 - Added support for decoding the ``globe-coordinate`` datatype.  [:pr:`28`]
 - Fixed a bug that raised :exc:`~urllib.error.HTTPError` when
   non-existent `Entity` was requested.  [:pr:`11`]
+- Added support for `time` datatypes with precision `9` (year-only).  [:pr:`26`]
```
Comment (dahlia) [inline docs/changes.rst:20]: This line is duplicated.

```suggestion
```

File: tests/datavalue_test.py [hunk 2]
```diff
@@ -155,6 +156,41 @@ def test_decoder_commonsMedia__string(fx_client: Client):
...
+            "amount": "+12",
+            "unit": "1"
+        },
+        'type': 'quantity'
+    })
+    gold = Quantity(
+        amount=12,
+        lower_bound=None,
+        upper_bound=None,
+        unit=None)
```
Comment (dahlia) [inline tests/datavalue_test.py:191]: Didn't you miss an `assert` statement to compare `decoded` with `gold`?

**Comment by nelson-liu** [comment]
thanks for catching that, must have messed up the merge.

**Comment by dahlia** [comment]
Thanks again!

**Pull request merged**